pact/Pact/Core/Serialise/CBOR_V1.hs

```diff
@@ -11,6 +11,9 @@ module Pact.Core.Serialise.CBOR_V1
   , encodeDefPactExec, decodeDefPactExec
   , encodeNamespace, decodeNamespace
   , encodeRowData, decodeRowData
+  -- used for
```

Comment:
used for?

Reply:
Agree, either this needs to be deleted, or fixed.